be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -594,12 +594,13 @@ bool FragmentExecutor::_is_in_colocate_exec_group(PlanNodeId plan_node_id) {
     return false;
 }
 
-void create_adaptive_group_initialize_events(RuntimeState* state, PipelineGroupMap&& unready_pipeline_groups) {
+static void create_adaptive_group_initialize_events(RuntimeState* state, WorkGroup* wg,
```

Comment:
is this function used by adaptive dop feature?

Reply:
Yes.